lambdas/session/src/main/java/uk/gov/di/ipv/cri/address/api/handler/SessionHandler.java

```diff
@@ -70,7 +70,7 @@ public APIGatewayProxyResponseEvent handleRequest(
 
         } catch (SessionValidationException e) {
 
-            eventProbe.log(INFO, e).counterMetric(EVENT_SESSION_CREATED, 0d);
+            eventProbe.log(ERROR, e).counterMetric(EVENT_SESSION_CREATED, 0d);
```

Comment:
I'd say this is still INFO and not an ERROR if we had an issue with the request

Reply:
I've never logged an exception with anything other than log level ERROR tbf. I'm not sure if the INFO log level provides the same level of verbosity as the ERROR level